### PR TITLE
feat(am): support alert engine configuration + user activity in helm charts for adaptive access

### PR DESCRIPTION
**Pull request by remisultan**
**Issue**

https://github.com/gravitee-io/issues/issues/8733

https://github.com/gravitee-io/issues/issues/7552
https://github.com/gravitee-io/issues/issues/7555
https://github.com/gravitee-io/issues/issues/7556

**Description**

Enables user activity in am helm charts

**Additional context**

https://github.com/gravitee-io/gravitee-access-management/pull/1882

### CODE REVIEW

**Review by leleueri — COMMENTED** [review]
Maybe we should add information about these settings into the README.adoc & README.md.
Other than that LGTM

**Review by leleueri — CHANGES_REQUESTED** [review]
~~If userManagement isn't define into the values.yaml, we will have an error~~

**Review by leleueri — APPROVED** [review]

**Review by leleueri — CHANGES_REQUESTED** [review]
Changelog isn't updated & chart.yaml

**Review by leleueri — CHANGES_REQUESTED** [review]
You have merge conflict.
Looks like, the next verison of the AM chart is `1.0.46` as currently we are in `1.0.45` https://artifacthub.io/packages/helm/graviteeio/am

File: am/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ This file documents all notable changes to [Gravitee.io Access Management 3.x](h
 ### 1.0.51
 
 - [X] Add upgrader framework job
```
Comment (leleueri) [inline am/CHANGELOG.md:7]: Does this feat (upgrader fwk), added in 1.0.51 ? :thinking:

File: am/Chart.yaml [hunk 2]
```diff
@@ -24,3 +24,5 @@ annotations:
   # https://artifacthub.io/packages/helm/graviteeio/am?modal=changelog
   artifacthub.io/changes: |
     - Add upgrader framework job
```
Comment (leleueri) [inline am/Chart.yaml:26]: Does this feat (upgrader fwk), added in 1.0.51 ? thinking

File: am/README.md [hunk 3]
```diff
@@ -333,6 +333,13 @@ httpClient:
 | ui.service.internalPortName | string | `http` |  |
 | ui.service.name | string | `"nginx"` |  |
 | ui.service.type | string | `"ClusterIP"` |  |
+| userManagement.activity.enabled | boolean | `false` | |
```
Comment (leleueri) [inline am/README.md:336]: options you added in the .adoc file to enable the connector are missing in this one

**Review by leleueri — CHANGES_REQUESTED** [review]

File: am/Chart.yaml (hunk 2 above)
Comment (remisultan) [reply am/Chart.yaml:26]: This was already on master

File: am/README.md (hunk 3 above)
Comment (remisultan) [reply am/README.md:336]: Will add them

File: am/CHANGELOG.md (hunk 1 above)
Comment (remisultan) [reply am/CHANGELOG.md:7]: https://github.com/gravitee-io/helm-charts/pull/262/files#r1050806000

**Review by leleueri — APPROVED** [review]

**Pull request merged**